scripts/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '2.3'
 services:
   db:
-    image: mariadb:10.4.17
+    image: mariadb:10.6
```

Comment:
what's the reason for this?

Reply:
mariabdb 4.17 was not working for my system

Reply:
that's ok, we just shouldn't change defaults in this sort of PR, it should be it's own PR so we can test it and discuss it, so let's remove it from this PR please.